yaml/xyz/openbmc_project/Logging/AuditLog.interface.yaml

```diff
@@ -0,0 +1,43 @@
+description: >
+    This interface provides methods to extract the Linux audit logged events.
+methods:
+    - name: ParseAuditLog
+      description: >
+          Parses all previously recorded log events into JSON format. Returns a
+          read-only file descriptor to the parsed file. Entries are sorted
+          oldest to newest.
+      returns:
+          - name: logFd
+            type: unixfd
+            description: >
+                The read-only file descriptor of the JSON formatted log events.
+      errors:
+          - xyz.openbmc_project.Common.Error.InternalFailure
+          - xyz.openbmc_project.Common.File.Error.Write
+          - xyz.openbmc_project.Common.File.Error.Open
+
+    - name: GetAuditLog
+      description: >
+          Parses a subset of previously recorded log events into JSON format.
+          Returns a read-only file descriptor to the parsed file. Entries are
+          sorted newest to oldest.
```

Comment:
why is this opposite of ParseAuditLog ?

Reply:
Note: I could easily make the limited number of entries method return the entries in the oldest to newest order instead, but still have it return the most recent entries up to the maxCount. The GUI would have to reverse the order for display in this case.

Please let me know if that is preferred.